DocumentService/Controllers/DocumentsController.cs

```diff
@@ -12,10 +14,23 @@ namespace DocumentService.Controllers
     [ApiController]
     public class DocumentsController : ControllerBase
     {
+        private IAzureBlobService _azureBlobService;
 
-        public DocumentsController()
+        public DocumentsController(IAzureBlobService azureBlobService)
         {
+            _azureBlobService = azureBlobService;
+        }
+
+        [HttpPost]
+        [Route("v1/documents/test")]
+        [ProducesResponseType(StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult UploadDocumentTest(IFormFile file)
+        {
+
+            var result = _azureBlobService.UploadFileAsync(file, AzureBlobContainer.DocumentsContainer).GetAwaiter().GetResult();
```

Comment:
```suggestion
            var result = this.azureBlobService.UploadFileAsync(file, AzureBlobContainer.DocumentsContainer).GetAwaiter().GetResult();
```

Reply:
Do we need "this" here, I feel it's better only in the constructor.